site/content/en/docs/concepts/multikueue.md

```diff
@@ -58,10 +58,8 @@ Known Limitations:
 There is an ongoing effort to overcome these limitations by adding the possibility to disable the reconciliation of some jobs by the Kubernetes `batch/Job` controller. Details in `kubernetes/enhancements` [KEP-4368](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/4368-support-managed-by-label-for-batch-jobs#readme).
 
 ### JobSet
-Known Limitations:
-- Since unsuspending a JobSet in the manager cluster will lead to its local execution and updating the status of a local JobSet could conflict with its main controller, you should only install the JobSet CRDs, but not the controller.
 
-An approach similar to the one described for [`batch/Job`](#batchjob) is taken into account to overcome this. 
+Since unsuspending a JobSet in the manager cluster will lead to its local execution and updating the status of a local JobSet could conflict with its main controller, MultiKueue expects the JobSets submitted to a ClusterQueue using it to have `alpha.jobset.sigs.k8s.io/managed-by` label set to `kueue`.
```

Comment:
```suggestion
MultiKueue expects the JobSets submitted to a ClusterQueue to have the `spec.managedBy` field set to `kueue.x-k8s.io/multikueue`.
```
Once we migrate to use the field, I would suggest `kueue.x-k8s.io/multikueue` as the reserved value, by analogy to k8s, where the value is `kubernetes.io/job-controller`.